README.md

```diff
@@ -9,83 +9,124 @@
+## API Documentation
 
-If you want to contribute your improvements, please follow the steps outlined in [our CONTRIBUTION guidelines](./CONTRIBUTE.md)
+For an easy and interactive way to test the api, visit our [API Playground](https://openrouteservice.org/dev/#/api-docs).
+After obtaining your key you can try out the different endpoints instantly and start firing requests.
+Go to the [API Reference](https://giscience.github.io/openrouteservice/api-reference/) for more information.
 
-The [sourcespy dashboard](https://sourcespy.com/github/giscienceopenrouteservice/) provides a high level overview of the repository including technology summary, module dependencies and other components of the system.
 
 ## Installation
 
-We suggest using docker to install and launch openrouteservice backend. In short, a machine with a working [docker installation](https://www.digitalocean.com/community/tutorial_collections/how-to-install-and-use-docker) will get everything done for you. 
+You can easily run openrouteservice yourself! We suggest using docker to install and launch openrouteservice. In short, a machine with a working [docker installation](https://www.digitalocean.com/community/tutorial_collections/how-to-install-and-use-docker) will get everything done for you. 
 
 Only use nightly (main branch) if you know what you do. We recommend running docker compose with the latest release version:
 
-```bash
-# For example for the latest release
-git clone https://github.com/GIScience/openrouteservice.git
-cd openrouteservice
-# Checkout latest version
-export LATEST_ORS_RELEASE=$(git describe --tags --abbrev=0); 
-git checkout $LATEST_ORS_RELEASE
-# If the docker folder exists cd into it
-cd docker || echo "No docker folder found. Continue with next step."
-# Now change the version the docker-compose.yml uses
-sed -i='' "s/openrouteservice\/openrouteservice:nightly/openrouteservice\/openrouteservice:$LATEST_ORS_RELEASE/g" docker-compose.yml
-sed -i='' "s/openrouteservice\/openrouteservice:latest/openrouteservice\/openrouteservice:$LATEST_ORS_RELEASE/g" docker-compose.yml
-# Run docker compose with
+Change to the directory where you want to install your local openrouteservice and first create some directories, where openrouteservice will persist its data:
+```shell
+mkdir -p ors-docker/config ors-docker/elevation_cache ors-docker/graphs ors-docker/files ors-docker/logs
+```
+
+Get the docker compose file for a release, e.g. v8.0.0: 
+```shell
+wget https://github.com/GIScience/openrouteservice/releases/download/v8.0.0/docker-compose.yml
+```
+
+Start openrouteservice in the background:
+```shell
 docker compose up -d
 ```
 
-For more details, check the [docker installation guide](https://giscience.github.io/openrouteservice/run-instance/running-with-docker).
+This will pull the openrouteservice docker image of the selected version and start it up using an example setup

```

Comment:
We should only name the preferred installation version here and link to docs for the rest